obsolete/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM gcr.io/google_appengine/base
+
+ENV MESSAGE \\n\
+********************************** NOTICE **********************************\\n\
+\\n\
+  The google/ruby images have been obsoleted.\\n\
+\\n\
+  If you want to deploy a Ruby application to Google App Engine, please use\\n\
```

Comment:
If you want to deploy a Ruby application to Google App Engine, you can simply specify `runtime: ruby` in your app.yaml.  If you'd like to extend this runtime, you can use gcr.io/google_appengine/ruby as your base image.